packages/contracts-bedrock/scripts/libraries/Executables.sol

```diff
@@ -31,4 +31,20 @@ library Executables {
         commands[2] = "cast abi-encode 'f(string)' $(git rev-parse HEAD || cat .gitcommit)";
         return abi.decode(Process.run(commands), (string));
     }
+
+    function prependContentToFile(string memory _contentToPrepend, string memory _outfile) public {
```

Comment:
This being `public` is why we have a linked library, let's always use internal for library methods
```suggestion
    function prependContentToFile(string memory _contentToPrepend, string memory _outfile) internal {
```

Reply:
Thanks @mds1 !